data_buffer.cpp

```diff
@@ -452,17 +452,19 @@ Vector2 DataBuffer::read_vector2(CompressionLevel p_compression_level) {
 }
 
 Vector2 DataBuffer::add_normalized_vector2(Vector2 p_input, CompressionLevel p_compression_level) {
+	const uint32_t is_not_zero = p_input.length_squared() > CMP_EPSILON;
```

Comment:
I don't like the name of this var, it's incorporating negation which makes it a bit harder to reason about any logic it's involved with.
It would be better to just reverse it's meaning and name to `is_zero` and use negation in the logic

Reply:
Normally I agree with you on this, but considering how it's used this variable later, I would need to negate all the logic; and that would be way more confusing.